<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>503</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="sideBarContainer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>-1</y>
         <width>691</width>
         <height>431</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="appTitleLabel">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>80</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier New</family>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Редактор структурованих нотаток</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchLineEdit">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Пошук...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="createNoteButton">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Створити нотатку</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources.qrc">
              <normaloff>:/icons/plus-circle.svg</normaloff>:/icons/plus-circle.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="manageSchemasButton">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Керувати схемами</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources.qrc">
              <normaloff>:/icons/edit.svg</normaloff>:/icons/edit.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="importNoteButton">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Імпорт</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources.qrc">
              <normaloff>:/icons/import.png</normaloff>:/icons/import.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="settingsButton">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Налаштування</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources.qrc">
              <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QComboBox" name="sortComboBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Спочатку нові</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Спочатку старі</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Назва (А-Я)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Назва (Я-А)</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="notesListWidget">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
